admin/backuppc/files/backuppc.init

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh /etc/rc.common
+
+START=95
+STOP=10
+
+USE_PROCD=1
+BACKUPPC_BIN="/usr/share/backuppc/bin/BackupPC"
+BACKUPPC_USER=backuppc
+
+# it would be better if it was possible to do this at install time, but we
+# can't, because in case of an openwrt image bundled with backuppc, all
+# ownerships other than root are lost.
+preconfigure() {
+    # create backuppc group and user if needed
+    source /lib/functions.sh
```

Comment:
This should be placed before START. See https://github.com/openwrt/packages/blob/master/utils/lxc/files/lxc-auto.init for an example. source throws a warning in shellcheck: SC2039: In POSIX sh, 'source' in place of '.' is undefined.

Reply:
done